### PR TITLE
#9: add controller file and update application model and route

### PR DESCRIPTION
**Pull request by sheidanouri**
add controller file and update application model and route
the authentication needs to be added

### CODE REVIEW

File: src/controllers/applicationController.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const Application = require('./applicationModel');
```
Comment (Younesnz) [inline src/controllers/applicationController.js:1]: Shule jan, the relative path should be `../models/applicationModel`

File: src/models/applicationModel.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
```
Comment (Younesnz) [inline src/models/applicationModel.js:2]: please add Joi  validation for user inputs. it should return the validation function

File: src/routes/applications.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const router = require('express').Router();
+const auth = require('./auth');  // assuming you have an auth middleware to authenticate users
```
Comment (Younesnz) [inline src/routes/applications.js:2]: we'll add authentication functionality on the next tasks. for now focus on CRUD operations

File: src/routes/applications.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+const applicationController = require('./applicationController');
```
Comment (Younesnz) [inline src/routes/applications.js:3]: the relative path should be `../controllers/applicationController`

**Review by Younesnz — CHANGES_REQUESTED** [review]

**Review by atakann — APPROVED** [review]

**Pull request closed without merging**